x-pack/legacy/plugins/monitoring/public/directives/main/index.html

```diff
@@ -1,3 +1,4 @@
+<div id="internal-monitoring-warning" style="margin: 12px;"></div>
```

Comment:
I suggest aligning this with the content. We usually use an 8px grid, therefore I guess this should do the trick

```suggestion
<div id="internal-monitoring-warning" style="margin: 16px 8px 16px;"></div>
```